packages/pyright-internal/src/languageServerBase.ts

```diff
@@ -1225,4 +1228,10 @@ export abstract class LanguageServerBase implements LanguageServerInterface {
             workspace.serviceInstance.recordUserInteractionTime();
         });
     }
+
+    protected getDocumentationUrlForDiagnosticRule(rule: string) {
```

Comment:
Should this always return a string, or would it be `string | undefined`? Looking at the usage above I'm not sure, but I can't see us ever returning nothing for a given rule.